### PR TITLE
Avoid logging passwords on user creation

### PR DESCRIPTION
**Pull request by CyberDem0n**
Fixes https://github.com/zalando/patroni/issues/1365

### CODE REVIEW

File: patroni/postgresql/bootstrap.py [hunk 1]
```diff
@@ -310,7 +310,13 @@ def create_or_update_role(self, name, password, options):
         CREATE ROLE "{0}" WITH {1};
     END IF;
 END;$$""".format(name, ' '.join(options))
-        self._postgresql.query(sql, *params)
+        self._postgresql.query('SET log_statement TO none')
+        self._postgresql.query("SET log_min_error_statement TO 'log'")
```
Comment (bradnicholson) [inline patroni/postgresql/bootstrap.py]: If someone set's `log_min_duration` low enough or to 0, it will still log, will it not?

Comment (CyberDem0n) [reply patroni/postgresql/bootstrap.py]: Good question, we need to check it

**Comment by RafiaSabih** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**